tests/swagger_generation/test_swagger_generator.py

```diff
@@ -238,6 +239,16 @@ def update_foo(foo_uid):
             multiple_authenticators.swagger_v3_generator,
             multiple_authenticators.EXPECTED_SWAGGER_V3,
         ),
+        (
```

Comment:
Bless you @barakalon for setting these tests up in such a delightfully extensible way 🏆  😃

Reply:
One gold star for me!

Reply:
![image](https://user-images.githubusercontent.com/31477753/125132588-626fd580-e0b9-11eb-97c2-85115f50d211.png)
You own a significant share of these too! 😁